tests/test_transplants.py

```diff
@@ -716,6 +716,60 @@ def test_integrated_transplant_simple_stack_saves_data_in_db(
     assert job.landed_revisions == {1: 1, 2: 2, 3: 3}
 
 
+def test_integrated_transplant_simple_partial_stack_saves_data_in_db(
+    db,
+    client,
+    phabdouble,
+    auth0_mock,
+    release_management_project,
+    register_codefreeze_uri,
+):
+    phabrepo = phabdouble.repo(name="mozilla-central")
+    user = phabdouble.user(username="reviewer")
+
+    # Create a stack with 3 revisions.
+    d1 = phabdouble.diff()
+    r1 = phabdouble.revision(diff=d1, repo=phabrepo)
+    phabdouble.reviewer(r1, user)
+
+    d2 = phabdouble.diff()
+    r2 = phabdouble.revision(diff=d2, repo=phabrepo, depends_on=[r1])
+    phabdouble.reviewer(r2, user)
+
+    d3 = phabdouble.diff()
+    r3 = phabdouble.revision(diff=d3, repo=phabrepo, depends_on=[r2])
+    phabdouble.reviewer(r3, user)
+
+    # Request a transplant, but only for the first revision in the stack.
```

Comment:
This comment does not match what this test is doing. Perhaps you meant to only include one revision, or this should say "first two revisions"?